### PR TITLE
career path

### PR DESCRIPTION
**Pull request by Femi-Benjamin**
Please check and confirm the Career path menu

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@emotion/react": "^11.10.5",
-    "@emotion/styled": "^11.10.5",
-    "@mui/material": "^5.11.8",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/material": "^5.11.11",
```
Comment (kelani34) [inline package.json:14]: Why is the dependency changing?

File: src/App.jsx [hunk 2]
```diff
@@ -15,6 +15,8 @@ import store from "./redux/store";
 import { Provider } from "react-redux";
 import Result from "./pages/Result";
 import Career from "./pages/Career";
+import Patth from "./pages/Patth";
+import DropDown from "./pages/DropDown";
```
Comment (kelani34) [inline src/App.jsx:19]: Please always use index.js in pages to fix you import

File: src/pages/Result.jsx [hunk 3]
```diff
@@ -2,8 +2,8 @@ import React from 'react'
 
 const Result = () => {
     return (
-        <div>
-            <h1 className='px-[10px] font-mulish not-italic font-bold md:text-[48px] text-[28px] md:leading-[60px] text-center mt-[87.73px] md:mb-[48px]'>You will succeed at a career path in.....</h1>
+        <div className='bg-[#E4E4E4] h-screen'>
```
Comment (kelani34) [inline src/pages/Result.jsx:5]: Please change the height. Its only needs to be h-full

File: src/pages/Patth.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { DropDownLayout } from '../layout'
+
+
+const Patth = () => {
+    return (
+        <div className='bg-[#E4E4E4] h-screen'>
+            <div className='flex justify-center items-center pt-[214px]'>
+                <div className='md:w-[657px] sm:w-auto md:h-[376.06px] sm:h-auto border sm:px-5 border-[#7C7E80] rounded-[7.11231px] bg-[#FFFFFF] text-center items-center'>
```
Comment (kelani34) [inline src/pages/Patth.jsx:9]: Please limit using sm: for media query. target the large screen- lg, the medium screen- md, then set a width for mobile

File: src/pages/Result.jsx [hunk 5]
```diff
@@ -2,8 +2,8 @@ import React from 'react'
...
+            <h1 className='px-[10px] font-mulish not-italic font-bold md:text-[48px] text-[28px] md:leading-[60px] text-center pt-[87.73px] md:mb-[48px]'>You will succeed at a career path in.....</h1>
             <div className=' md:px-[400px] px-[30px] py-[30px]'>
```
Comment (kelani34) [inline src/pages/Result.jsx:7]: Paddings are too much px-[400px] looks horrible on tablet or mobile screens. please stick to flex

Comment (kelani34) [reply src/pages/Result.jsx:7]: Pull from main I just fixed this

**Review by kelani34 — CHANGES_REQUESTED** [review]

File: src/pages/Patth.jsx (hunk 4 above)
Comment (Femi-Benjamin) [reply src/pages/Patth.jsx:9]: Noted

File: src/pages/Result.jsx (hunk 3 above)
Comment (Femi-Benjamin) [reply src/pages/Result.jsx:5]: Height changed to Full

File: src/App.jsx (hunk 2 above)
Comment (Femi-Benjamin) [reply src/App.jsx:19]: I tried it here did not work

**Review by kelani34 — APPROVED** [review]

**Pull request merged**